.gitignore

```diff
@@ -58,3 +58,4 @@ docs/_build/
 
 # IDE
 .idea
+.vscode
```

Comment:
you might want to put this in your `~/.gitignore` as that saves you from adding it to every project.